libs/community/langchain_community/embeddings/optimum_intel.py

```diff
@@ -0,0 +1,161 @@
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+import torch
+from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import BaseModel, Extra
+from optimum.intel import IPEXModel
```

Comment:
If optimum.intel is not installed you can't load this module. You need to protect this code with try catch and with comments on how to install the dependencies. Look at other files on how to use 3rd party libraries.